style.css

```diff
@@ -288,3 +310,196 @@ form textarea {
   margin-top: 30px;
   margin-bottom: 5px;
 }
+
+/* Medium devices (landscape tablets, 768px and up) */
+@media only screen and (min-width: 768px) {
+  #menu-icon {
+    display: none;
+  }
+
+  a {
+    text-decoration-line: none;
+    color: #344563;
+  }
+
+  .container {
+    padding-right: 10%;
+    padding-left: 10%;
+  }
+
```

Comment:
- Hey, let's correctly align the form button to the center to match the design as much as we can. see the images below:

*Figma design*
![Screenshot from 2023-02-22 16-59-06](https://user-images.githubusercontent.com/59397159/220682290-2c654f2f-8adc-4d14-be07-aebd897dd338.png)

*Your design*
![Screenshot from 2023-02-22 16-58-46](https://user-images.githubusercontent.com/59397159/220682308-2e8d09b9-f2c9-456c-b3d6-ee3a1c5c9f3e.png)

Reply:
Thank you! I haven't noticed that!

Reply:
@KUBAHO3, It happens, we forget at times!